.env

```diff
@@ -2,7 +2,7 @@ PORT=8001
 FLASK_DEBUG=true
 DEVEL=true
 ADVANTAGE_API=https://contracts.canonical.com/
-DATABASE_URL=sqlite:///usn.sqlite3
+DATABASE_URL=sqlite:///
```

Comment:
Any reason we are changing this? is quite useful to persist on disk so we have the same data in between re-runs.

Reply:
Ok, I thought this was the right way to do it. So should i keep it as `sqlite:///usn.sqlite3`?